performance_test/src/communication_abstractions/resource_manager.cpp

```diff
@@ -17,6 +17,11 @@
 #include <memory>
 #include <string>
 
+#ifdef PERFORMANCE_TEST_OPENDDS_ENABLED
+using namespace OpenDDS::RTPS;
```

Comment:
I generally like to avoid `using namespace` if possible to avoid having unexpected name collisions

Reply:
done